CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-# Change Log
+# Changelog
```

Comment:
`standard-version` when it creates the changelog spells it as two words.

Reply:
I know, but as far as I know this is the correct spelling and the latest release of `standard-version` fixed a bug that inserted the title with any new release. We shouldn't have problems with that in the future.

Reply:
It was the latest version of standard-version that created this updated changelog file.. with the top level heading...

Reply:
I've tested in locally and it worked fine. The title wasn't changed.